test/cypress/plugins/stubs.js

```diff
@@ -0,0 +1,261 @@
+'use strict'
+
+// Local dependencies
+const userFixtures = require('../../fixtures/user_fixtures')
+const gatewayAccountFixtures = require('../../fixtures/gateway_account_fixtures')
+
+module.exports = {
+  getUserSuccess: (opts = {}) => {
+    const aValidUserResponse = userFixtures.validPasswordAuthenticateResponse(opts).getPlain()
```

Comment:
we use the fixtures to setup the stubs.
the aim will be that these fixtures are satisfactorily validated by also being used to generate stubs, which will have different data, but the same structure